connector/manifest.xml

```diff
@@ -42,6 +42,7 @@
       <customization name="CAP_JDBC_METADATA_READ_PRIMARYKEYS" value="no"/>
       <customization name="CAP_JDBC_QUERY_CANCEL" value="no"/>
       <customization name="CAP_JDBC_SUPPRESS_ENUMERATE_SCHEMAS" value="yes"/>
+      <customization name="CAP_JDBC_USE_ADAPTIVE_FETCH_SIZE" value="no"/>
```

Comment:
Modifying this capability here.  To avoid these lines being logged after every query:
```
ERROR com.tableausoftware.jdbc.FetchSizeHelpers - Failed to update fetch size.
java.sql.SQLFeatureNotSupportedException: com.mongodb.jdbc.MongoResultSet.setFetchSize(MongoResultSet.java:535)
```